### PR TITLE
BAH-2962_token | Created a script to run on sms-service startup.

### PR DESCRIPTION
**Pull request by anubhavBeehyv**
- Created a script to run on sms-service startup.
- Script generate a private and public key and a token which will be sent to openmrs via mounting.

### CODE REVIEW

**Comment by angshu** [comment]
@mohan-13 maybe you want to check the entry point script? also the dockerfile. Should we use yum or use apt?

**Comment by anubhavBeehyv** [comment]
> @mohan-13 maybe you want to check the entry point script? also the dockerfile. Should we use yum or use apt?

apt was not found in the container ..while yum commands were working fine .

**Comment by mohan-13** [comment]
> > @mohan-13 maybe you want to check the entry point script? also the dockerfile. Should we use yum or use apt?
> 
> apt was not found in the container ..while yum commands were working fine .

Yup, amazoncorretto base images comes with yum as the package manager. So that should be fine.

File: src/main/java/org/bahmni/sms/web/SMSController.java [hunk 1]
```diff
@@ -4,26 +4,40 @@
...
+    private static final Logger logger = LogManager.getLogger(DefaultSmsSender.class);
 
     @RequestMapping(method = RequestMethod.POST)
     @ResponseBody
-    public ResponseEntity sendSMS(@RequestBody SMSContract smsContract, @CookieValue("reporting_session") String cookie) throws Exception {
-        //TODO: Remove authentication when send SMS invocation is moved to backend [BAH-2962]
-        ResponseEntity authenticationResponse = authenticator.authenticate(cookie);
-        if (authenticationResponse.getStatusCode().is2xxSuccessful())
+    public ResponseEntity sendSMS(@RequestBody SMSContract smsContract, @RequestHeader(value = HttpHeaders.AUTHORIZATION, required = false) String authorizationHeader) throws Exception {
+        boolean isValidToken = false;
```
Comment (mohan-13) [inline src/main/java/org/bahmni/sms/web/SMSController.java]: Shouldn't we leverage Spring Security config which would intercept every request and do authorisation? Adding it in controller may not be a scalable solution in future if we add one more endpoint. @angshu thoughts?

Comment (anubhavBeehyv) [reply src/main/java/org/bahmni/sms/web/SMSController.java]: Added spring security please review this again

**Review by angshu — APPROVED** [review]

**Review by mohan-13 — APPROVED** [review]

**Pull request merged**